src/AVFoundation/Enums.cs

```diff
@@ -157,6 +157,15 @@ public enum AVAudioSessionInterruptionFlags : nuint_compat_int {
 		ShouldResume = 1
 	}
 
+	[Flags]
+	[NoWatch, NoTV]
+	[Availability (Deprecated = Platform.iOS_6_0)]
+	[Native]
+	public enum  AVAudioSessionSetActiveFlags : uint
```

Comment:
is it needed by some API ?
we don't support iOS < 6 (and Apple does not allow this into the App Store anymore) so I don't see the value of binding it today